client/extensions/auto-tagger/src/auto-tagging.tsx

```diff
@@ -285,21 +291,23 @@ export function getAutoTaggingComponent(superdesk: ISuperdesk, label: string): I
                 name: _title,
                 description: newItem.description,
                 source: 'manual',
+                creator: "Human",
+                relevance: 0.47,
```

Comment:
just wondering shouldn't this be actually `1.0` when picked by a journalist?

Reply:
After some consideration, Paul has decided that human relevance score should be .47. 
Thank you for pointing out though.